From 812439e0ae085a37238dc97f2921c732a962d222 Mon Sep 17 00:00:00 2001
From: Stefan Sayer <stefan.sayer@googlemail.com>
Date: Mon, 29 Oct 2012 20:03:21 +0100
Subject: [PATCH] b/f: sst: use proper delimiter for supported list

support for SST in the remote would not be understood properly in some cases

thanks to Jon Bonilla for reporting
---
 core/plug-in/session_timer/SessionTimer.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/plug-in/session_timer/SessionTimer.cpp b/core/plug-in/session_timer/SessionTimer.cpp
index a91668c..eafd9fa 100644
--- a/core/plug-in/session_timer/SessionTimer.cpp
+++ b/core/plug-in/session_timer/SessionTimer.cpp
@@ -271,7 +271,7 @@ void SessionTimer::updateTimer(AmSession* s, const AmSipRequest& req) {
     
     remote_timer_aware = 
       key_in_list(getHeader(req.hdrs, SIP_HDR_SUPPORTED, SIP_HDR_SUPPORTED_COMPACT),
-		  TIMER_OPTION_TAG, true);
+		  TIMER_OPTION_TAG);
     
     // determine session interval
     string sess_expires_hdr = getHeader(req.hdrs, SIP_HDR_SESSION_EXPIRES,
-- 
1.7.10.4

